website/docs/docs/dbt-cloud-apis/migrating-to-v2.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Migrating to dbt Cloud Administrative API v2"
+description: "You should migrate to API v2 while we deprecate API v4 "
+sidebar_label: "Migrating to API v2"
+id: "migrating-to-v2"
+---
+
```

Comment:
Should we include a blurb here about why we're deprecating V4 and asking them to step down to V2 (maybe even why there is no V3)? Security teams might see reverting to a lower version as a red flag.